x-pack/plugins/security_solution/public/detections/components/response_actions/osquery/osquery_response_action.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiCode, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { ResponseActionValidatorRef } from '../response_actions_form';
+import type { ArrayItem } from '../../../../shared_imports';
+import { useKibana } from '../../../../common/lib/kibana';
+import { NOT_AVAILABLE, PERMISSION_DENIED, SHORT_EMPTY_TITLE } from './translations';
+
+interface IProps {
+  item: ArrayItem;
+  formRef: React.RefObject<ResponseActionValidatorRef>;
+}
+
+// eslint-disable-next-line react/display-name
+export const OsqueryResponseAction = React.memo((props: IProps) => {
```

Comment:
Adding the following should get rid of the lint error and avoid adding a eslint disable.

```
OsqueryResponseAction.displayName = "OsqueryResponseAction";
```

Reply:
👍